src/migrator/sync.ts

```diff
@@ -25,10 +24,10 @@ export const postRun = async (tx: Tx, model: ApiRootModel): Promise<void> => {
 
 	const modelName = model.apiRoot;
 
-	const exists = await checkModelAlreadyExists(tx, modelName);
+	const exists = await sbvrUtils.isModelNew(tx, modelName);
 	if (!exists) {
 		(sbvrUtils.api.migrations?.logger.info ?? console.info)(
-			'First time executing, running init script',
+			`First time executing ${modelName}, running init script`,
```

Comment:
I find putting the variable part in quotes can help make it clear that you're quoting some user input
```suggestion
			`First time executing '${modelName}', running init script`,
```